tests/rest_api_tests.py

```diff
@@ -1,7 +1,9 @@
 """Unit Tests for Rest API."""
 
+import sys
 import unittest
-import rest_api
+sys.path.append('../')   # allows for importing of files not in the same directory | pylint: disable=line-too-long
```

Comment:
could we move this in the init.py ? Actually do we even need it now ?

Reply:
I moved it to the init.py and fixed all the test files to work with the new modules.